parser.js

```diff
@@ -274,7 +274,7 @@
     function createClassRange(min, max, from, to) {
       // See 15.10.2.15:
       if (min.codePoint > max.codePoint) {
-        throw SyntaxError('invalid range in character class');
+        bail('invalid range in character class: ' + min.raw + '-' + max.raw, from, to);
```

Comment:
I had originally thought about something like this but was worried that it could be confusing in light of the already symbol heavy nature of regular expressions. Whenever I read a quoted regular expression that has any sort of escaping my mind has to do some gymnastics to figure out what its intent is.

Can make the change with or without escaping of the content, but personally I found unescaped with normal content to be the clearest.